src/runtime/node/buffer/$cloudflare.ts

```diff
@@ -1,7 +1,7 @@
 // https://nodejs.org/api/buffer.html
 import type nodeBuffer from "node:buffer";
 
-export { INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
+export { atob, btoa, INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
 
 import { INSPECT_MAX_BYTES, resolveObjectURL } from "./index";
```

Comment:
you would need to add `atob, btoa` here too I think?
But you can also use workerd impl